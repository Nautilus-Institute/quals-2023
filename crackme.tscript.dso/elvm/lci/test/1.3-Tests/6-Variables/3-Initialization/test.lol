HAI 1.3
	I HAS A var1 ITZ 1
	I HAS A var2 ITZ 2.345
	I HAS A var3 ITZ "Lorem ipsum dolor sit"
	VISIBLE var1 var2 var3
KTHXBYE
