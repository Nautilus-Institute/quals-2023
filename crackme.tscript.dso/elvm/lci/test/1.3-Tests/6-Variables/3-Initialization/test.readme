This test checks that variables may be initialized when declared.
