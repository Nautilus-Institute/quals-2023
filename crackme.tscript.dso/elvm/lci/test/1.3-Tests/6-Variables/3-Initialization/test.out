12.34Lorem ipsum dolor sit
